dev/NavigationView/NavigationView.idl

```diff
@@ -108,6 +108,20 @@ runtimeclass NavigationViewBackRequestedEventArgs
 {
 }
 
+[WUXC_VERSION_MUXONLY]
+[webhosthidden]
+runtimeclass NavigationViewExpandingEventArgs
+{
+    NavigationViewItemBase ExpandingItemContainer { get; };
```

Comment:
>NavigationViewItemBase [](start = 4, length = 22)

It's a little wierd that we are expanding NavigationViewItem but the container is base here. 'void Expand(NavigationViewItem item)'
Maybe we need to make it consistent.

Reply:
This was addressed in a similar comment that you left previously. It got marked as 'outdated' after I pushed new updates. The reason why we are leaving this as NavigationViewItemBase is for future-proofing purposes (in case there is a new menuitem that is added which will also have the capability to have children).